doc/source/conf.py

```diff
@@ -74,11 +73,12 @@ def setup(app):
     '{exc_repo}': 'obs://openSUSE:Leap:15.1/standard',
     '{exc_kiwi_repo}':
         'obs://Virtualization:Appliances:Builder/openSUSE_Leap_15.1',
-    '{schema_version}': '7.1'
+    '{schema_version}': '7.1',
+    '{kiwi}': 'KIWI-NG'
```

Comment:
From our native speaker, she would replace the dash with a space. Additionally, I would also define the full name, without the "NG" acronym:

```suggestion
    '{kiwi}': 'KIWI NG',
    '{kiwi-full}': 'KIWI Next Generation',
```